### ISSUE TITLE
Verification email never got sent

### ISSUE THREAD

**Issue by Bardivan** [jeremylarner/keyteki#377]
Im trying to log in but it says i need to verfy my email but the verification email never went through.

**Comment by TheCrowDreamer** [comment]
I have same problem

**Comment by howmuchwood89** [comment]
Any luck on this? I am having the same problem.

**Comment by Bardivan** [comment]
Yes you need to join the discord server for keyforge and go to the
development section. post your having verification troubles and someone
will should help you
